docs/attribution.mdx

```diff
@@ -19,134 +19,250 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
+- Gilchrist County data from [Florida Department of Revenue](https://pointmatch.floridarevenue.com/General/AddressFiles.aspx). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-03-29.
+- Glades County data from [Florida Department of Revenue](https://pointmatch.floridarevenue.com/General/AddressFiles.aspx). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-03-30.
+- Gulf County data from [Florida Department of Revenue](https://pointmatch.floridarevenue.com/General/AddressFiles.aspx). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-03-31.
+- Hamilton County data from [Florida Department of Revenue](https://pointmatch.floridarevenue.com/General/AddressFiles.aspx). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-04-01.
+- [Hardee County](https://gis.hardeecounty.net/arcgis/rest/services/Addressing/MapServer/3). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-04-02.
+- [Hendry County](https://gis.hendryfla.net/datasets/a84275b4f69d468a883d4715c98c29bc/about). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-04-03.
+- [Hernando County](https://server.hernandopa-fl.us/server/rest/services/Basemap/MapServer/1). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-04-04.
+- [Highlands County](https://hcbcc.maps.arcgis.com/home/item.html?id=f20fc360487d4b73a55bf5930a988c53). Available under [Public domain by judicial decision](https://www.myfloridalegal.com/ag-opinions/records-license-agreements-for-county-maps). Distributed by OpenAddresses. Accessed: 2025-04-05.
 - [Hillsborough County](https://gis2017-01-10t133755357z-hillsborough.opendata.arcgis.com/datasets/hillsborough::siteaddresspoint/about). Available under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/). Distributed by OpenAddresses. Accessed: 2025-02-02.
```

Comment:
Should we standardize the Hillsborough source, Manatee source, and any other existing FL and CA sources to also reference the Public domain stance we're taking? I think that list of counties would be:

California
- City and County of San Francisco
- Fresno County
- Los Angeles County
- Napa County
- Nevada County
- Sacramento County
- Santa Cruz County

Florida
- Hillsborough County
- Manatee County

Reply:
I'm open to either way, but was planning to just leave them as an explicit permissive license is probably better than an implied one.